docs/bucket/notifications/README.md

```diff
@@ -1,7 +1,11 @@
 # Minio Bucket Notification Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
 
-Changes in a bucket, such as object uploads and removal, can be monitored using bucket event notification
-mechanism and can be published to the following targets:
+Events occurring on objects in a bucket can be monitored using bucket event notifications. Event types supported by Minio server are `Put`, `Post`, `Copy`, `CompleteMultipartUpload`, `Delete`, `Get` and `Head`.
```

Comment:
events have a consistent format like `s3:ObjectCreated:Put` would it make sense to mention that here instead?  - the list can be found [here](http://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html#notification-how-to-event-types-and-destinations)

Reply:
we can mention that, but I felt that is lower level detail which may not be useful for user - for example how is `s3:ObjectCreated:Put` set via mc or minio-go (they have their own ways of setting notification). 

IMO adding `s3:ObjectCreated:Put` will be difficult to interpret for user.

Reply:
Since this is a server we are really talking about API details here. So we need to mention that IMO.  Canned names would only confuse we have to double down to indicate that canned is only for `mc`.

Some one using minio-go, aws-sdk you would want them to know this detail.

Reply:
> Since this is a server we are really talking about API details here

In that case makes sense to add the details. Updated the PR.